spec/regression/logistics/model/delivery.graphqls

```diff
@@ -38,7 +38,6 @@ type Delivery @rootEntity(flexSearch: true, flexSearchLanguage: DE) @roles(readW
 
     description: String @flexSearchFulltext(includeInSearch: true)
     enumFlexSearch: Foobarit @flexSearch
-    aNumber: Int @flexSearch(includeInSearch: true)
```

Comment:
you could leave in the test but just remove `includeInSearch`. DeliveryItem.quantity still exists, but that's in a child entity